common/test/services/IndexPageTest.scala

```diff
@@ -48,20 +48,16 @@ import scala.concurrent.Future
         fail("Wrong type (expected: IndexPage, real: Result)")
       case Some(page) =>
         val front = IndexPage.makeFront(page, edition)
-        front.containers.length should be(1)
+        front.containers should not be empty
```

Comment:
@jfsoul and I just chatted about this. These tests aren't very good. At the moment they have to be rewritten with each data file change and that shouldn't be the case. Anyway this change is fine for now. We will look at it when we get a chance 👍